lib/taurus/core/tango/tangodevice.py

```diff
@@ -185,9 +185,14 @@ def getDisplayDescrObj(self, cache=True):
         ret = []
         for name, value in desc_obj:
             if name.lower() == u'device state' and self.stateObj is not None:
-                tg_state = self.stateObj.read(cache).rvalue.name
-                value = u"%s (%s)" % (value, tg_state)
-            ret.append((name, value))
+                try:
+                    tg_state = self.stateObj.read(cache).rvalue.name
+                    value = u"%s (%s)" % (value, tg_state)
+                except DevFailed as e:
+                    error = str(e).replace(u"\n", u"<br/>")
+                    template = u"""%s <font style="white-space:pre">%s</font>"""
+                    value = template % (value, error)
+                ret.append((name, value))
```

Comment:
It looks like in your commit, you moved the `ret.append` inside the `if` block while originally it was in the outer `for` block (i.e. you added one indentation)

Was this on purpose? If so, why?

Reply:
It was just a mistake. Thanks for the review.